src/ai/backend/manager/models/minilang/queryfilter.py

```diff
@@ -50,6 +51,20 @@
 FieldSpecType: TypeAlias = Mapping[str, FieldSpecItem] | None
 
 
+def enum_field_getter(enum_cls: Type[Enum]) -> Callable[[str], Enum]:
+    def get_enum(value: str) -> Enum:
+        try:
+            return enum_cls[value]
+        except KeyError:
+            try:
+                return enum_cls(value)
+            except ValueError:
+                enum_names = ", ".join([e.value for e in enum_cls])
+                raise ValueError(f"expected one of `{enum_names}`, got `{value}`")
```

Comment:
A very minor suggestion to disambiguate the commas:
```suggestion
                raise ValueError(f"expected one of `{enum_names}`; got `{value}`")
```